src/workers/postUpdated.ts

```diff
@@ -606,6 +613,12 @@ const fixData = async ({
   }
 
   const duration = data?.extra?.duration / 60;
+  const i18n: PostI18n = {};
+  const titleI18n = data?.meta?.translate_title?.translations;
```

Comment:
regarding my previous question, i see that data.meta is stored as is, which means we store twice the titles. can save it only once?
We can consider virtual generated column
https://www.postgresql.org/docs/current/ddl-generated-columns.html

Reply:
yeah, it is duplicated currently

Reply:
psql only supports stored generated columns, but maybe we can do it on typeorm side or delete it from `meta` when saving

Reply:
why do we need psql? am I missing something?
Ideally I want to avoid duplication as it's a lot of data

Reply:
you referenced psql generated columns above, and they can't be virtual so if we use them it will be the same as if we are storing double data

I agree on duplication, we need to avoid it

Reply:
Nope it is not supported.

Reply:
We synced offline, basically in 7b4f3b3c5360d128912c851f39b595cf9e95a473 I refactored so it is loaded from `contentMeta`

I also had to add support in graphorm for `requiredColumns` configuration so I can only load part of the `jsonb` field instead of loading all `contentMeta` whenever resolving post entity.